db/sg_replicate_cfg.go

```diff
@@ -1396,9 +1403,16 @@ func (m *sgReplicateManager) GetReplicationStatus(replicationID string, options
 			if err != nil {
 				return nil, err
 			}
-			status = &ReplicationStatus{
-				ID:     replicationID,
-				Status: remoteCfg.TargetState,
+			if remoteCfg.AssignedNode == "" {
+				status = &ReplicationStatus{
+					ID:     replicationID,
+					Status: ReplicationStateError,
```

Comment:
I'm wondering if it's worth considering an extra state (`ReplicationStateUnassigned`) for this case?

Presumably the error is recoverable when a node running sg-replicate becomes available, which I think would differ from existing error states.